[2023-09-04 20:31:32,595] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:31:32] "[35m[1mPOST /modelo HTTP/1.1[0m" 500 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:31:56,087] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.138:5000 (Press CTRL+C to quit) - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:00,189] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:00] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:00,285] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:00] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:00,291] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:00] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:00,292] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:00] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:00,390] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:00] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:00,397] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:00] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:06,396] ERROR log_exception() L1449 Exception on /modelo [POST] - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask_openapi3\scaffold.py", line 191, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\app.py", line 251, in add_modelo
    modelo = Modelo(
       ^^^^^^^^^^^^^
TypeError: __init__() got an unexpected keyword argument 'nom_modelo'
[2023-09-04 20:32:06,400] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:06] "[35m[1mPOST /modelo HTTP/1.1[0m" 500 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:42,224] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.138:5000 (Press CTRL+C to quit) - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:46,484] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:46] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:46,574] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:46] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:46,583] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:46] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:46,584] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:46] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:46,638] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:46] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:46,646] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:46] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:32:50,672] ERROR log_exception() L1449 Exception on /modelo [POST] - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask_openapi3\scaffold.py", line 191, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\mvp\puc-rio-mvp-sprint-3-component-C-2\app.py", line 251, in add_modelo
    modelo = Modelo(
       ^^^^^^^^^^^^^
TypeError: __init__() got an unexpected keyword argument 'cod_marca'
[2023-09-04 20:32:50,676] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:32:50] "[35m[1mPOST /modelo HTTP/1.1[0m" 500 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:33:14,596] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:33:14] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:33:14,617] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:33:14] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:33:14,620] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:33:14] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:33:14,621] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:33:14] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:33:14,675] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:33:14] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-04 20:33:14,678] INFO _log() L224  127.0.0.1 - - [04/Sep/2023 20:33:14] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\mvp\puc-rio-mvp-sprint-3-component-C-2\vm\Lib\site-packages\werkzeug\_internal.py L224 
